app/Gists/File.php

```diff
@@ -0,0 +1,64 @@
+<?php namespace Gistlog\Gists;
+
+use Illuminate\Contracts\Support\Arrayable;
+
+class File implements Arrayable
+{
+    public $name;
+    public $type;
+    public $language;
+    public $url;
+    public $size;
+    public $content;
+
+    /**
+     * @param array|ArrayAccess $rawFile
+     * @return File
+     */
+    public static function fromGitHub($rawFile)
+    {
+        $file = new self;
+
+        $file->name = $rawFile['filename'];
+        $file->type = $rawFile['type'];
+        $file->language = $rawFile['language'];
+        $file->url = $rawFile['raw_url'];
+        $file->size = $rawFile['size'];
+        $file->content = $rawFile['content'];
+
+        return $file;
+    }
+
+    /**
+     * @param array|ArrayAccess $rawFiles
+     * @return Collection
+     */
+    public static function multipleFromGitHub($rawFiles)
+    {
+        // Remove the blog post itself.
+        array_shift($rawFiles);
```

Comment:
As I'm thinking through this, it might be handy to link potential users to an example gistlog style gist and encourage them to fork it. This would act as a template and would include a `blog.md` file as well as a `gistlog.yml` file listing out all of the options available to them.